cpp/test/SuperMarketTest.cpp

```diff
@@ -22,6 +25,9 @@ TEST(SuperMarketTest, foo) {
         teller.addSpecialOffer(SpecialOfferType::TenPercentDiscount, toothbrush, 10.0);
 
         Receipt receipt = teller.checksOutArticlesFrom(cart);
+        ReceiptPrinter rp = ReceiptPrinter();
+        
+        ApprovalTests::Approvals::verify(rp.printReceipt(receipt));
```

Comment:
Could you remove these three lines? They are part of the solution, and the starting point shouldn't include them. I'd like to replace them with equivalent assert statements like the ones you'll find in the csharp and java versions.